dev-ruby/ffi-compiler/files/respect-cflags.patch

```diff
@@ -0,0 +1,17 @@
+diff --git a/lib/ffi-compiler/compile_task.rb b/lib/ffi-compiler/compile_task.rb
+index 363c247..637fd3d 100644
+--- a/lib/ffi-compiler/compile_task.rb
++++ b/lib/ffi-compiler/compile_task.rb
+@@ -26,9 +26,9 @@ module FFI
+         @libraries = []
+         @headers = []
+         @functions = []
+-        @cflags = DEFAULT_CFLAGS.dup
+-        @cxxflags = DEFAULT_CFLAGS.dup
+-        @ldflags = DEFAULT_LDFLAGS.dup
++        @cflags = ENV['CFLAGS'].split || DEFAULT_CFLAGS.dup
```

Comment:
I would like to avoid these kinds of patches to keep closer to upstream and to avoid surprises when people approach this code from the Ruby eco-system side. It's probably best to coordinate these things with upstream instead and see if they are open for this kind of change.

Note that this code is also broken: it will break when no such environment variable is provided.

Reply:
Upstream is dead for ffi-compiler, otherwise I would have done that.  Sorry, I don't know enough Ruby to get this correct then - how should this be in order to work as intended?

Reply:
Fixed the patch.  My apologies on this.

Reply:
The most recent change upstream is 4 months ago, so I'm not sure this should be considered dead.